Database/Corrections/QuestieQuestFixes.lua

```diff
@@ -566,5 +567,18 @@ function QuestieQuestFixes:Load()
             [QuestieDB.questKeys.exclusiveTo] = {5887,5888,5889,5890,},
             [QuestieDB.questKeys.specialFlags] = 0,
         },
+        [7670] = { -- #1432
+        [QuestieDB.questKeys.name] = "Lord Grayson Shadowbreaker",
+        [QuestieDB.questKeys.startedBy] = {{5149},nil,nil,},
+        [QuestieDB.questKeys.finishedBy] = {{928},nil,},
+        [QuestieDB.questKeys.requiredLevel] = 60,
+        [QuestieDB.questKeys.questLevel] = 60,
+        [QuestieDB.questKeys.requiredRaces] = 5,
```

Comment:
On [Wowhead](https://classic.wowhead.com/quest=7670/lord-grayson-shadowbreaker) it is marked as an alliance quest an therefore it should be:

```suggestion
        [QuestieDB.questKeys.requiredRaces] = 77,
```